ansible/playbooks/dataloader/opensearchbackup.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Opensearch backup
+  hosts: loader
+  connection: local
+  gather_facts: yes
+
+  tasks:
+    #install collections
+    - name: install aws collection
+      shell: "ansible-galaxy collection install {{ item | quote }} --force"
+      loop:
+        - community.docker:<=3.4.3
+        - git+https://github.com/CBIIT/datacommons-devops.git#/ansible/collections/bento/common/,opensearch_snapshot
+
```

Comment:
the branch reference here will need to be updated - once the merge is complete this branch should be deleted